src/MouseTooltip.jsx

```diff
@@ -28,10 +35,18 @@ class MouseTooltip extends React.PureComponent {
   }
 
   getTooltipPosition = ({ clientX: xPosition, clientY: yPosition }) => {
+    const screenWidth = window.innerWidth;
+    const screenHeight = window.innerHeight;
```

Comment:
Can you destructure these properties?

```suggestion
    const { innerWidth: screenWidth, innerHeight: screenHeight } = window;
```